src/types/transactions/base.ts

```diff
@@ -366,4 +366,9 @@ export interface TransactionParams {
    * All new Algorand accounts are participating by default. This means they earn rewards.
    */
   nonParticipation?: boolean;
+
+  /**
+   * Int representing extra pages of memory to rent during an application create transaction.
```

Comment:
Could you mention that a page is 1024 bytes, like in https://github.com/algorand/py-algorand-sdk/blob/3ca47273258ae99e168272f95b66416359a00c97/algosdk/future/transaction.py#L1161?